components/array-object-table/package.json

```diff
@@ -26,7 +26,7 @@
   "author": "Steve Sims",
   "contributors": ["Gavin Orland"],
   "license": "MIT",
-  "homepage": "https://www.npmjs.com/package/@govuk-frederic/array-object-table",
+  "homepage": "https://github.com/stevesims/govuk-frederic/tree/master/components/array-object-table#readme",
```

Comment:
Do these links really need the `#readme` bit?

Reply:
[Apparently](https://docs.npmjs.com/files/package.json#homepage) that's the way it's usually done. Could remove it though if you prefer.

Reply:
I don't think they should link to the readme TBH, the package location is perfect as it is the homepage for the component.

Reply:
Okedokie - will search and replace and push this up shortly.